docs/server/source/production-deployment-template/template-kubernetes-azure.rst

```diff
@@ -99,7 +99,7 @@ Finally, you can deploy an ACS using something like:
    --master-count 3 \
    --agent-count 2 \
    --admin-username ubuntu \
-   --agent-vm-size Standard_D2_v2 \
+   --agent-vm-size Standard_L4s \
```

Comment:
When I looked at the list of Azure VM sizes on the web page https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-sizes-specs , I couldn't find any occurrence of the string `L4`